tests/rados/rados_test_util.py

```diff
@@ -123,3 +123,23 @@ def get_device_path(host, osd_id):
     ][0]
     should_not_be_empty(dev_path, "Failed to get device path")
     return dev_path
+
+
+def get_slow_requests_log(installer, start, end):
+    """
+    Retrieve slow op requests from installer
```

Comment:
Can we add docstrings on the params sent, esp the formats in which `start and `end` need to be provided.

Reply:
will be adding the doc string.